components/icons/.editorconfig

```diff
@@ -0,0 +1,27 @@
+root = true
```

Comment:
I'm not sure we need this editorconfig in this package, we should just use the editorconfig  from the root